spec/knackhq/client_spec.rb

```diff
@@ -0,0 +1,206 @@
+require 'spec_helper'
+
+describe Knackhq::Client do
+  let(:client) { knack_client }
+  let(:base_uri) { 'https://api.knackhq.com/v1' }
+  let(:x_knack_application_id) { '123456789' }
+  let(:x_knack_rest_api_key) { '123-456-789' }
+
+  let(:knack_client) do
+    Knackhq::Client.new(base_uri,
+                        x_knack_application_id,
+                        x_knack_rest_api_key)
+  end
+
+  describe '#new' do
+    it { expect(knack_client).to be_a Knackhq::Client }
+  end
+
+  describe '#request.get' do
```

Comment:
I'm still not fond of this :)

Will calling request alone not produce this behavior or does it need to have an actual request made?